README.md

```diff
@@ -118,13 +118,47 @@ Fetch a single vector tile from the tileset `id` by an `x`,`y`,`z` coordinate.
 
 ### Sync
 
-#### [TODO] `GET /sync/wifi`
+#### `GET /sync/targets`
 
-some kind of sync thing. it's going to be great.
+Returns list of available sync targets. Right now, only lists other services broadcasting on the local network through mdns using the 'mapeo-sync' key.
```

Comment:
What does the returned data look like? An array? What's in each array slot, etc?

Reply:
Does hitting this route trigger a query, or is it happening in the background and this is the cached view of what targets are available?

Reply:
It's happening in the background and this is a cached view of what targets are available.